src/IssueViz.UnitTests/TableControls/VSTableEventProcessorTests.cs

```diff
@@ -131,48 +130,25 @@ public void SelectionChanged_SingleItem_TryGetValueReturnsTrueButValueIsNotIssue
 
             SimulatePostProcessEvent();
 
-            CheckMonitorCalledOnce(null);
+            CheckSelectionServiceCalledOnce(null);
 
             SanityCheckSnapshotAccessedOnce();
         }
 
         [TestMethod]
```

Comment:
Simplified the two tests: passing now simply `Mock.Of<IAnalysisIssueVisualization>()`